tests/performance/scripts/download.sh

```diff
@@ -23,6 +23,8 @@ dataset_paths["hits10"]="https://clickhouse-private-datasets.s3.amazonaws.com/hi
 dataset_paths["hits100"]="https://clickhouse-private-datasets.s3.amazonaws.com/hits_100m_single/partitions/hits_100m_single.tar"
 dataset_paths["hits1"]="https://clickhouse-datasets.s3.amazonaws.com/hits/partitions/hits_v1.tar"
 dataset_paths["values"]="https://clickhouse-datasets.s3.amazonaws.com/values_with_expressions/partitions/test_values.tar"
+dataset_paths["tpch1"]="https://clickhouse-datasets.s3.amazonaws.com/h/1/tpch.tar"
+dataset_paths["tpch10"]="https://clickhouse-datasets.s3.amazonaws.com/h/10/tpch.tar"
 
```

Comment:
Quick question! Are we benchmarking ClickHouse in out-of-memory situations? SF10 seems too small to achieve this.

Reply:
The performance tests need to run quickly (< 5 sec), and as a result, they need to use small datasets, and because of that they cannot test OOMs.

Reply:
I mostly mean if the tables don't fit in memory. That happens sometimes?

Reply:
I am testing OOM situations with the `system.zeros` and `system.numbers` tables: https://github.com/ClickHouse/ClickHouse/issues/74750 Thanks for the info.